src/components/Brands.tsx

```diff
@@ -24,6 +24,7 @@ const brands = [
 
 export const Brands = () => (
   <section className="mb-8 w-full bg-shorebirdBg3 py-12 sm:py-24 lg:mb-16 lg:mt-16">
+    <div className="absolute -top-16" id="brands" />
```

Comment:
What is this div for? Just spacing? Does it need an id?

Reply:
This is the same div that pricing uses.  Just making https://shorebird.dev/#brands work.